src/securityPolicy.c

```diff
@@ -476,24 +484,27 @@ security_policy_t policyForSignTxCertificateStaking(
 		ASSERT(false);
 	}
 
-	if (stakingKeyPath) {
-		DENY_UNLESS(bip44_isOrdinaryStakingKeyPath(stakingKeyPath));
+	if (stakeCredential->type == STAKE_CREDENTIAL_KEY_PATH) {
+		DENY_UNLESS(bip44_isOrdinaryStakingKeyPath(&stakeCredential->pathSpec));
```

Comment:
I'd move `switch (certificateType)` to the beginning of the function, and then add this check to `case SIGN_TX_USECASE_ORDINARY_TX`.

Reply:
Good call, multisig doesn't have pathSpec.